Description: Adapt to apt 1.9.11
 The individual hash classes are gone, replaced with Hashes
 class.
Author: Julian Andres Klode <juliank@ubuntu.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2020-02-27

--- libqapt-3.0.4.orig/src/debfile.cpp
+++ libqapt-3.0.4/src/debfile.cpp
@@ -27,7 +27,7 @@
 
 #include <apt-pkg/debfile.h>
 #include <apt-pkg/fileutl.h>
-#include <apt-pkg/md5.h>
+#include <apt-pkg/hashes.h>
 #include <apt-pkg/tagfile.h>
 
 #include <QDebug>
@@ -193,13 +193,13 @@ QByteArray DebFile::md5Sum() const
 {
     FileFd in(d->filePath.toStdString(), FileFd::ReadOnly);
     debDebFile deb(in);
-    MD5Summation debMD5;
+    Hashes debMD5(Hashes::MD5SUM);
 
     in.Seek(0);
 
     debMD5.AddFD(in.Fd(), in.Size());
 
-    return QByteArray(debMD5.Result().Value().c_str());
+    return QByteArray(debMD5.GetHashString(Hashes::MD5SUM).HashValue().c_str());
 }
 
 QStringList DebFile::fileList() const
